_includes/nav.html

```
@@ -1,6 +1,12 @@
-{% assign en_url = page.url | replace_first: '/es', '/en' | replace_first: '/ca', '/en' %}
-{% assign es_url = page.url | replace_first: '/en', '/es' | replace_first: '/ca', '/es' %}
-{% assign ca_url = page.url | replace_first: '/es', '/ca' | replace_first: '/en', '/ca' %}
+{% assign ca_url = page.url %}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)